distributed-databases/cloudnative-postgres/examples/storageclass.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: ebs-sc
+provisioner: ebs.csi.aws.com
+volumeBindingMode: WaitForFirstConsumer
+allowVolumeExpansion: true
+parameters:
+  csi.storage.k8s.io/fstype: xfs
```

Comment:
Is `xfs` recommended for PostgreSQL? If Yes then add this details to Website doc